apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerClientCreationOptionsTest.cs

```diff
@@ -137,5 +137,14 @@ public async Task GetCredentialsAsync_GoogleCredential()
             var options = new SpannerClientCreationOptions(builder);
             Assert.NotNull(await options.CreateSpannerClientAsync(new Spanner.V1.SpannerSettings()));
         }
+
+        [Fact]
+        public async Task Equality_UniverseDomainAndEndpoint()
```

Comment:
nit: move this with the other equality tests a few lines up.

Reply:
And also complete the test similar to the ones up there including what the effects of setting universe domain have for the endpoint here.